### PR TITLE
[plugin.video.regiotv@matrix] 0.1.2+matrix.1

### PR DESCRIPTION
**Pull request by dagwieers**
### Add-on details:

- **General**
  - Add-on name: Regio TV
  - Add-on ID: plugin.video.regiotv
  - Version number: 0.1.2+matrix.1
  - Kodi/repository version: matrix

- **Code location**
  - URL: https://github.com/add-ons/plugin.video.regiotv
  
The Regio TV add-on lists all available Flemish regional television stations.

Available channels: ATV, BRUZZ, Focus TV, ROB-tv, TVL, TV Oost, WTV

[I]This add-on is not endorsed by any television station, and is provided 'as is' without any warranty of any kind.[/I]

### Description of changes:

v0.1.2 (2020-08-01)
- Add support for IPTV Manager
- Add support for referer
- Add RegioTV artwork

v0.1.1 (2020-02-11)
- Fix regional channels BRUZZ, Focus TV and WTV

v0.1.0 (2019-11-08)
- Initial public release
  

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.video.regiotv/resources/settings.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<settings>
+
+  <category label="30860"> <!--Integration -->
+    <setting label="30861" type="lsep"/> <!-- Integration with other add-ons -->
+    <!-- IPTV Manager -->
+    <!-- setting label="30875" help="30876" type="action" action="InstallAddon(service.iptv.manager)" option="close" visible="!System.HasAddon(service.iptv.manager)"/ --> <!-- Install IPTV Manager add-on -->
+    <setting label="30877" help="30878" type="bool" id="iptv.enabled" default="true" visible="String.StartsWith(System.BuildVersion,18) + System.HasAddon(service.iptv.manager) | System.AddonIsEnabled(service.iptv.manager)" />
```
Comment (basrieter) [inline plugin.video.regiotv/resources/settings.xml:8]: Where are all these settings used? I cannot seem to find it?

File: plugin.video.regiotv/resources/lib/data.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+# GNU General Public License v3.0 (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+''' A list of static data '''
+
+from __future__ import absolute_import, division, unicode_literals
+
+CHANNELS = [
```
Comment (basrieter) [inline plugin.video.regiotv/resources/lib/data.py:7]: Are all these URL's freely available? And if so, where?

File: plugin.video.regiotv/resources/lib/addon.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+    listing = []
+    for channel in CHANNELS:
+        is_folder = False
+        is_playable = True
+
+        label = '{name} [COLOR gray]| {label}[/COLOR]'.format(**channel)
+        plot = '[B]{name}[/B]\nRegio {label}\n\n[I]{description}[/I]\n\n[COLOR yellow]{website}[/COLOR]'.format(**channel)
+        list_item = ListItem(label=label, label2=channel.get('label'), offscreen=True)
+        list_item.setProperty(key='IsInternetStream', value='true' if is_playable else 'false')
```
Comment (basrieter) [inline plugin.video.regiotv/resources/lib/addon.py]: I don't think that we know such a property for ListItems? @enen92

**Review by basrieter — COMMENTED** [review]
Please check my comments.

File: plugin.video.regiotv/resources/lib/addon.py (hunk 3 above)
Comment (enen92) [reply plugin.video.regiotv/resources/lib/addon.py]: I am not sure we should allow this to be enforced from the add-on. This is an internal property that Kodi sets depending on the stream. Better to set the mimetype or find why Kodi doesn't set it for the stream and either fix Kodi or explain why this is needed

File: plugin.video.regiotv/resources/settings.xml (hunk 1 above)
Comment (dagwieers) [reply plugin.video.regiotv/resources/settings.xml:8]: It is used by IPTV Manager to determine the add-on is supported.
https://github.com/add-ons/service.iptv.manager/wiki/Design#advertise-from-settingsxml

File: plugin.video.regiotv/resources/lib/data.py (hunk 2 above)
Comment (dagwieers) [reply plugin.video.regiotv/resources/lib/data.py:7]: This was asked in the previous review as well. The URLs are available from each website.
https://github.com/xbmc/repo-plugins/pull/2587#issuecomment-557798049

File: plugin.video.regiotv/resources/lib/addon.py (hunk 3 above)
Comment (dagwieers) [reply plugin.video.regiotv/resources/lib/addon.py]: Removed.

**Review by enen92 — APPROVED** [review]

**Pull request merged**